dev/conductor/core/lib/src/codesign.dart

```diff
@@ -41,6 +41,10 @@ class CodesignCommand extends Command<void> {
         processManager = checkouts.processManager {
     if (framework != null) {
       _framework = framework;
+      _framework!.upstreamRemote = const Remote(
```

Comment:
This doesn't seem like the right place to fix (it's a red flag that we're removing the final from upstreamRemote).

It seems like this is a test only fix in https://cs.opensource.google/flutter/flutter/+/master:dev/conductor/core/test/codesign_integration_test.dart;l=47?q=CodesignCommand&ss=flutter%2Fflutter to add the upstreamRemote

Reply:
umm right I wanted to override upstream remote without removing the final.

umm my understanding is that the cause of error was that initially we have a frameworkRepository whose upstream url was set to `file:///opt/s/w/ir/x/w/flutter sdk/`. And then it got passed down to https://cs.opensource.google/flutter/flutter/+/master:dev/conductor/core/lib/src/codesign.dart;l=42, but codesign.dart was not able to override the upstream url at line 42 (so it kept the passed down frameworkRepository), and it eventually gets cloned into the error of `Cloning framework from file:///opt/s/w/ir/x/w/flutter sdk/ to /opt/s/w/ir/x/w/flutter_conductor_checkouts/framework...`. umm I wonder if there is a way to override upstream (and like Casey said ideally without having to remove the final keyword)